### PR TITLE
Fix links to client libraries

### PR DESCRIPTION
**Pull request by dmend**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-250acba668/docs/cloud-keep/v1/developer-guide/)

**Comment by dmend** [comment]
Hi @meker12, when you get a chance, could you please review this?  It's a pretty small change.

File: api-docs/getting-started/send-request-ovw.rst [hunk 1]
```diff
@@ -5,24 +5,22 @@ Send requests to the API
...
 
-.. note:: 
-     You can also use Rackspace Cloud API services by using the following methods: 
+.. note::
+     You can also use Rackspace Cloud API services by using the following methods:
 
-     -  If you are developing applications or automation, try using `Rackspace SDKs`_, the
-        `Rackspace CLI`_, or `OpenStack client applications`_.
+     -  If you are developing applications or automation, try using the
+        `OpenStack python-barbicanclient`_ library.
```
Comment (meker12) [inline api-docs/getting-started/send-request-ovw.rst]: Tiny fix -- add space between *barbican* and *client*.

Comment (dmend) [inline api-docs/getting-started/send-request-ovw.rst]: The actual name of the library is python-barbicanclient, but I agree that it's not very readable.  I'll use an english description instead.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-a74f6bb88d/docs/cloud-keep/v1/developer-guide/)

**Pull request merged**